addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/ParamMinerAPI.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.paramminer;
+
+import net.sf.json.JSONObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.zaproxy.zap.extension.api.ApiAction;
+import org.zaproxy.zap.extension.api.ApiException;
+import org.zaproxy.zap.extension.api.ApiImplementor;
+import org.zaproxy.zap.extension.api.ApiResponse;
+import org.zaproxy.zap.extension.api.ApiResponseElement;
+
+public class ParamMinerAPI extends ApiImplementor {
+    private static final String PREFIX = "paramMiner";
+
+    private static final String ACTION_HELLO_WORLD = "helloWorld";
+
+    private static final Logger LOGGER = LogManager.getLogger(ParamMinerAPI.class);
+
+    public ParamMinerAPI() {
+        this.addApiAction(new ApiAction(ACTION_HELLO_WORLD));
+    }
+
+    @Override
+    public String getPrefix() {
+        return PREFIX;
+    }
+
+    @Override
+    public ApiResponse handleApiAction(String name, JSONObject params) throws ApiException {
+        switch (name) {
+            case ACTION_HELLO_WORLD:
+                LOGGER.debug("hello world called");
```

Comment:
Not that it really matters but since you included it for now.
`LOGGER.debug(Constant.messages.getString("paramminer.api.action.helloWorld"));`